bndtools.core.test/.metadata/.plugins/org.eclipse.core.runtime/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,9 @@
+content_assist_proposals_background=255,255,255
```

Comment:
We should not be checking in the `.metadata/` folder!?

Reply:
Oops...